src/main/ipc.js

```diff
@@ -28,7 +28,6 @@ function init (cwd) {
   ipc.on('setAspectRatio', (e, ...args) => main.setAspectRatio(...args))
   ipc.on('setBounds', (e, ...args) => main.setBounds(...args))
   ipc.on('setProgress', (e, ...args) => main.setProgress(...args))
-  ipc.on('setTitle', (e, ...args) => main.setTitle(...args))
```

Comment:
Skipped this message, since we can take care of this directly in render. Also, the backend should decide the window title, just like any state.